nvm.sh

```diff
@@ -660,6 +661,39 @@ nvm_checksum() {
   fi
 }
 
+# For iojs, with a view to supporting SHA256 for Node > 0.10 soon as well.
+# https://github.com/creationix/nvm/pull/664
+nvm_checksum_256() {
+  local NVM_CHECKSUM
+
+  if nvm_has "sha256sum" && ! nvm_is_alias "sha256sum"; then
```

Comment:
Yeah, might help narrow it down a bit. Theoretically, if Travis is running Linux it should be using ` sha256sum `, unless they've done something unusual with their build boxes.